mailpoet/lib/Config/Migrator.php

```diff
@@ -664,6 +665,21 @@ private function sqlify($model, $attributes) {
     return implode("\n", $sql);
   }
 
+  private function updateNewsletterIdCanBeNullInSendingQueues() {
```

Comment:
Just to be sure — we need to do this manually here because the delta DB will not detect that change?